docker/Dockerfile

```diff
@@ -137,7 +137,7 @@ FROM base-build-cuda as build-kokkos-cuda
 WORKDIR /opt/kokkos-${KOKKOS_VERSION}
 # RUN wget --progress=dot:giga https://github.com/kokkos/kokkos/archive/refs/tags/${KOKKOS_VERSION}.tar.gz
 # RUN tar -zxf ${KOKKOS_VERSION}.tar.gz && mv /opt/kokkos-${KOKKOS_VERSION}/kokkos-${KOKKOS_VERSION}/* /opt/kokkos-${KOKKOS_VERSION}
-RUN git clone --branch fix-rtld-deepbind https://github.com/cz4rs/kokkos.git /opt/kokkos-${KOKKOS_VERSION}
+RUN git clone --branch lightning https://github.com/vincentmr/kokkos.git /opt/kokkos-${KOKKOS_VERSION}
```

Comment:
Are you planning to maintain a fork for this long-term?

Reply:
No, this is just to pin a commit for which GPU backends work on all tested platforms (e.g. ISAIC, Perlmutter, LUMI). It appears `cz4rs` rebased his branch or something and the `fix-rtld-deepbind` branch does not work anymore. Hope I can get some time to dig into this in the near future.

Reply:
Ah, sounds good. Ok fine by me.